tikv-client/src/main/java/com/pingcap/tikv/types/YearType.java

```diff
@@ -83,7 +83,7 @@ protected void encodeProto(CodecDataOutput cdo, Object value) {
 
   @Override
   public ExprType getProtoExprType() {
-    return ExprType.Year;
+    return ExprType.Uint64;
```

Comment:
Why we need to change this to `UInt64`?

Reply:
year no longer exists in ExprType. Uint64 is its replacement.

Reply:
Year has changed back to Int64. ICYMI @Novemser